prow/gubernator/.gitignore

```diff
@@ -0,0 +1,3 @@
+.coverage
```

Comment:
How about creating fork on `kyma-incubator` with Gubernator changes? Right now is it hard to determine what changes did you make. Also, as you said it is only temporary.

Take also in mind that we are cloning this repository in almost every job.

Reply:
This is a very good idea! I'll update the PR in this way.